upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -106,6 +106,16 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 		}
 		dest["FlannelBackendType"] = func() string { return flannelBackendType }
 	}
+
+	if tf.cluster.Spec.Networking != nil && tf.cluster.Spec.Networking.AmazonVPC != nil {
+		defaultPluginImageName := "602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:0.1.4"
```

Comment:
Setting customizable default values here and in nodeup doesn’t really look right to me. I’d personally move the default and customizabikity to the kops cluster spec networking portion.

Reply:
@chrisz100 - Fair point! I'll take a look.